src/tests.ts

```diff
@@ -8,8 +8,8 @@ import {
 } from './index';
 
 assert.strictEqual(
-    addQueryToUrl({ url: 'http://foo.com/' })({ queryToAppend: { a: 'b' } }),
-    'http://foo.com/?a=b',
+    addQueryToUrl({ url: 'http://foo.com/' })({ queryToAppend: { a: 'b', number: 1 } }),
```

Comment:
Wanna add a boolean here maybe too? And an `undefined` value actually as well would be a nice guarantee.